addons/skin.estuary/xml/DialogPVRChannelsOSD.xml

```diff
@@ -37,7 +37,7 @@
 							<width>80</width>
 							<height>70</height>
 							<aspectratio align="right">keep</aspectratio>
-							<texture>$INFO[listitem.icon]</texture>
+							<texture fallback="$INFO[Listitem.Icon]">$INFO[Listitem.EpgEventIcon]</texture>
```

Comment:
makes no sense, see my other comment.

Reply:
In this screen the Icon fallback is not active, we have to have it.

Reply:
Again, this is nonsense.

Reply:
Have you tested this or are you just guessing?